ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -36,7 +36,7 @@ Install RHODS
       IF  "${TEST_ENV}" in "${SUPPORTED_TEST_ENV}" and "${INSTALL_TYPE}" == "CLi"
              Install RHODS In Self Managed Cluster Using CLI  ${cluster_type}     ${image_url}
       ELSE IF  "${TEST_ENV}" in "${SUPPORTED_TEST_ENV}" and "${INSTALL_TYPE}" == "OperatorHub"
-          ${file_path} =    Set Variable    tasks/Resources/RHODS_OLM/install/
+          ${file_path} =    Set Variable    %{WORKSPACE=.}/ods-ci/ods_ci/tasks/Resources/RHODS_OLM/install/
```

Comment:
this will fail on our CI/CD because it changes directory to ods_ci before running.

I'm not against this change, but it must be done in parallel on CI/CD

Reply:
@bdattoma can u point me where we need to change this as well? to understand the full picture